src/features/shared-link-settings-modal/AllowDownloadSection.js

```diff
@@ -35,23 +37,35 @@ const AllowDownloadSection = ({
         </div>
     );
 
+    const tooltipMessage = { ...messages.directDownloadBlockedByAccessPolicy };
+    const downloadWrapperClass = isDirectLinkUnavailableDueToAccessPolicy ? 'bdl-is-disabled' : null;
+
     if (isDownloadAvailable) {
         return (
-            <div>
+            <div className={downloadWrapperClass}>
                 <hr />
-                <Fieldset title={<FormattedMessage {...messages.allowDownloadTitle} />}>
-                    <Checkbox
-                        isChecked={isDownloadEnabled}
-                        isDisabled={!canChangeDownload}
-                        label={<FormattedMessage {...messages.allowDownloadLabel} />}
-                        name="isDownloadEnabled"
-                        onChange={onChange}
-                        {...downloadCheckboxProps}
-                    />
-                    {(isDirectLinkAvailable || isDirectLinkUnavailableDueToDownloadSettings) && isDownloadEnabled
-                        ? directLinkSection
-                        : null}
-                </Fieldset>
+                <Tooltip
+                    isDisabled={!isDirectLinkUnavailableDueToAccessPolicy}
+                    text={<FormattedMessage {...tooltipMessage} />}
+                    position="middle-left"
+                >
+                    <Fieldset
+                        disabled={isDirectLinkUnavailableDueToAccessPolicy}
+                        title={<FormattedMessage {...messages.allowDownloadTitle} />}
+                    >
+                        <Checkbox
+                            isChecked={isDownloadEnabled}
```

Comment:
I think this is confusing, if the the download is restricted, should it still show as checked?

Reply:
`isDownloadEnbaled`is received as false if restricted.